src/main.zig

```diff
@@ -440,6 +440,8 @@ const usage_build_generic =
     \\  -dynamic                       Force output to be dynamically linked
     \\  -static                        Force output to be statically linked
     \\  -Bsymbolic                     Bind global references locally
+    \\  --compress-debug-sections=     Compress DWARF debug sections
```

Comment:
I believe we are still missing implementation for `build-*` subcommands. It seems to me that we only handle presence of this flag when invoking `cc` or  `c++` only.

Reply:
I am not sure how to do it. Can you point me to a such linker parameter that is accepted in `build-*` and the passed to the linker?

Reply:
Here's how it's done for `-search_paths_first` MachO linker flag for `build-*` https://github.com/ziglang/zig/blob/08459ff1c21d546c55e2ae954126e121ee88972e/src/main.zig#L927-L931

Reply:
Thanks, added. Also verified the arg is passed with `ZIG_VERBOSE_LINK=1` when building a trivial elf executable with `zig build-exe`.